src/components/UniversalLink/index.js

```diff
@@ -47,7 +47,7 @@ const UniversalLink = ({
 }
 
 UniversalLink.propTypes = {
-  as: PropTypes.node,
+  as: PropTypes.oneOfType([PropTypes.string, PropTypes.elementType]),
```

Comment:
Doesn't `node` already includes these ? 

```
 // Anything that can be rendered: numbers, strings, elements or an array
  // (or fragment) containing these types.
  optionalNode: PropTypes.node,
```

Reply:
Yes I thought so but I still get the warning with node, not with elementType :(